java/main/com/google/privacy/differentialprivacy/ConfidenceInterval.java

```diff
@@ -0,0 +1,17 @@
+package com.google.privacy.differentialprivacy;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * Stores the upper and lower bounds of a confidence interval.
+ */
+@AutoValue
+public abstract class ConfidenceInterval{
+    public abstract double getLowerBound();
```

Comment:
lowerBound()

Reply:
for consistency with other auto-value classes

Reply:
Should be good now.